test/commons/math/get-offset.js

```diff
@@ -1,104 +1,66 @@
-describe('getOffset', function () {
+describe('getOffset', () => {
   'use strict';
-  var fixtureSetup = axe.testUtils.fixtureSetup;
-  var getOffset = axe.commons.math.getOffset;
-  var round = 0.2;
+  const fixtureSetup = axe.testUtils.fixtureSetup;
+  const getOffset = axe.commons.math.getOffset;
+  const round = 0.2;
 
-  // Return the diagonal of a square of size X, or rectangle of size X * Y
-  function getDiagonal(x, y) {
-    y = typeof y === 'number' ? y : x;
-    return Math.sqrt(Math.pow(x, 2) + Math.pow(y, 2));
-  }
-
-  it('returns with + spacing for horizontally adjacent elms', function () {
-    var fixture = fixtureSetup(
-      '<a style="width:30px; margin-right:10px; display:inline-block">&nbsp;</a>' +
-        '<b style="width:20px; display:inline-block">&nbsp;</b>'
-    );
-    var nodeA = fixture.children[0];
-    var nodeB = fixture.children[1];
-    assert.closeTo(getOffset(nodeA, nodeB), 40, round);
-    assert.closeTo(getOffset(nodeB, nodeA), 30, round);
-  });
-
-  it('returns closest horizontal distance for elements horizontally aligned', function () {
-    var fixture = fixtureSetup(
-      '<a style="width:30px; height:30px; margin-right:10px; display:inline-block">&nbsp;</a>' +
-        '<b style="width:20px; height:20px; top:5px; position:relative; display:inline-block">&nbsp;</b>'
-    );
-    var nodeA = fixture.children[0];
-    var nodeB = fixture.children[1];
-    assert.closeTo(getOffset(nodeA, nodeB), getDiagonal(40, 5), round);
-    assert.closeTo(getOffset(nodeB, nodeA), 30, round);
+  it('returns center to edge of circle when both are undersized', () => {
+    const fixture = fixtureSetup(`
+      <button style="width: 10px; height: 10px; margin: 0; position: absolute; top: 0; left: 0">&nbsp;</button>
+      <button style="width: 10px; height: 10px; margin: 0; position: absolute; top: 50px; left: 0">&nbsp;</button>
+    `);
+    const nodeA = fixture.children[1];
+    const nodeB = fixture.children[3];
+    assert.closeTo(getOffset(nodeA, nodeB), 38, round);
```

Comment:
It's because the distance between the the centers of both buttons gets [subtracted by the circle radius](https://github.com/dequelabs/axe-core/pull/4117/files#diff-7056fdbaecbbd8da82e47e708803e085002edc843821a06fa5930c68514f201fR33) of 12 (50 - 12).